src/test/java/tests/ContractTests.java

```diff
@@ -0,0 +1,81 @@
+package tests;
+
+import base.BaseTest;
+import io.qameta.allure.Epic;
+import io.qameta.allure.Feature;
+import microservices.contract.models.Contract;
+import microservices.contract.models.DataContract;
+import microservices.contract.steps.ContractSteps;
+import microservices.token.models.DataToken;
+import microservices.token.models.Token;
+import microservices.token.steps.TokenSteps;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import util.CreateParameters;
+
+import java.net.HttpURLConnection;
+import java.util.ArrayList;
+import java.util.Map;
+
+@Epic("cardano")
+@Feature("api-contract")
+public class ContractTests extends BaseTest {
+    private ContractSteps contractSteps = new ContractSteps();
+    private Contract contract = new Contract();
+    private ArrayList<DataContract> dataContracts;
+
+    @Test(description = "verify that get list contract successfully", groups={"contract"},dataProvider = "paramSuccess")
+    public void getListParamSuccess(Object page, Object size, String sort){
+        Map<String, Object> param = new CreateParameters()
+                .withPage(page)
+                .withPageSize(size)
+                .withSort(sort)
+                .getParamsMap();
+        contractSteps.getListContracts(param)
+                .validateResponse(HttpURLConnection.HTTP_OK);
+
+        contract = contractSteps.saveResponseListContract();
```

Comment:
should be contract = (Contract) contractSteps.getListContracts(param)
                .validateResponse(HttpURLConnection.HTTP_OK). .saveResponseObject(Contract.class);